hydrolib/core/io/rr/network/models.py

```diff
@@ -0,0 +1,107 @@
+from typing import Callable, List, Optional
+
+from pydantic.class_validators import root_validator
+from pydantic.fields import Field
+
+from hydrolib.core.basemodel import BaseModel, FileModel
+from hydrolib.core.io.rr.network.parser import NodeFileParser
+from hydrolib.core.io.rr.network.serializer import NodeFileSerializer
+
+# Dictionary with `nt` mapped against the expected `mt`.
+nodetypes = {
+    43: 1,
+    44: 2,
+    45: 3,
+    46: 4,
+    None: 5,
+    34: 6,
+    35: 6,
+    47: 6,
+    48: 8,
+    49: 9,
+    50: 10,
+    51: 11,
+    52: 12,
+    56: 14,
+    55: 15,
+    54: 16,
+    69: 23,
+}
+
+
+class Node(BaseModel):
+    """Represents a node from the topology node file."""
+
+    id: str = Field(alias="id")
+    name: str = Field(alias="nm")
```

Comment:
apologies, I just found out 'nm' is **optional**.

Reply:
No problem, adjusted it :)